src/Appointment/Request.js

```diff
@@ -0,0 +1,328 @@
+import React, {useState} from 'react';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import './Request.css';
+
+class Request extends React.Component {
```

Comment:
I am fixing the linting issues here. But there should not be new line after every line.